k8s/namespaces/probate/probate-frontend/demo.yaml

```diff
@@ -8,7 +8,7 @@ spec:
     nodejs:
       ingressClass: traefik-no-proxy
       ingressHost: probate.demo.platform.hmcts.net
-      image: hmctspublic.azurecr.io/probate/frontend:pr-1560-38985c1-20220405162616 #{"$imagepolicy": "flux-system:demo-probate-frontend"}
```

Comment:
change should be like
 
     image: hmctspublic.azurecr.io/probate/frontend:pr-1560-38985c1-20220405162616 #{"$imagepolicy": "flux-system:probate-frontend"}